.env.sample

```diff
@@ -1,4 +1,5 @@
 DISCORD_CLIENT_ID=1082239786162606080
 NEXT_PUBLIC_DISCORD_CLIENT_ID=1082239786162606080
 DISCORD_CLIENT_SECRET=123
-DB_PASSWORD=password
+DATABASE_URL= postgres://{user}:{password}@{hostname}:{port}/{database-name}
```

Comment:
Are the curly braces here intentional?

Reply:
@timmyichen, thoughts on simplifying the code and just using `DATABASE_URL` in all environments? We can throw it at the bottom of the `.env` and automatically build it using the separate vars. Heroku can then overwrite `DATABASE_URL` all they want and the separate vars would simply no longer be used/referenced anywhere. eg.

```ini
...
DB_PASSWORD=password

# Configuration Values# Configuration Values
DB_PORT=5432
DB_USER=postgres
DB_HOST=localhost
DB_NAME=test

...

DATABASE_URL= postgres://$DB_USER:$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_NAME
```

[Var Expansion in NextJS Docs](https://nextjs.org/docs/basic-features/environment-variables#:~:text=automatically%20expand%20variables%20(%24VAR))

Reply:
yeah, it was, but I can change it?

Reply:
I guess as a whole we can probably remove it since it'll never be used in development

Reply:
gotcha!